lib/simp/rake/helpers/assets/rpm_spec/simp6.spec

```diff
@@ -238,7 +328,10 @@ URL:       %{lua: print(module_source)}
 BuildRoot: %{_tmppath}/%{package_name}-%{version}-%{release}-buildroot
 BuildArch: noarch
 
-Requires(pre,preun,post,postun): simp-adapter >= 0.0.1
+Requires(pre): simp-adapter >= 0.0.1
+Requires(preun): simp-adapter >= 0.0.1
+Requires(preun): simp-adapter >= 0.0.1
+Requires(postun): simp-adapter >= 0.0.1
 
 %if ("%{package_name}" != "pupmod-simp-simplib") && ("%{package_name}" != "pupmod-puppetlabs-stdlib")
 Requires: pupmod-simp-simplib >= 1.2.6
```

Comment:
Now that the requires are pulled from the metadata.json, this block of logic should be removed.